pkg/operator/webhooksupportabilitycontroller/degraded_webhook_admission.go

```diff
@@ -0,0 +1,69 @@
+package webhooksupportabilitycontroller
+
+import (
+	"context"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+func (c *webhookSupportabilityController) updateMutatingAdmissionWebhookConfigurationDegraded(ctx context.Context) v1helpers.UpdateStatusFunc {
+	condition := operatorv1.OperatorCondition{
+		Type:   MutatingAdmissionWebhookConfigurationDegradedType,
+		Status: operatorv1.ConditionUnknown,
+	}
+	webhookConfigurations, err := c.mutatingWebhookLister.List(labels.Everything())
+	if err != nil {
+		condition.Message = err.Error()
+		return v1helpers.UpdateConditionFn(condition)
```

Comment:
this won't put the operator into a degraded state, is that okay?

Reply:
Yes. This is typical behavior.